oktoberfest/ce_calibration.py

```diff
@@ -114,8 +127,7 @@ def _load_rawfile(self):
             pass
         else:
             raise ValueError(f"{switch} is not supported as rawfile-type")
-        print(self.raw_path)
-        self.raw_path = self.raw_path.as_posix().replace(".raw", ".mzml")
+        self.raw_path = Path(self.raw_path).as_posix().replace(".raw", ".mzml")
```

Comment:
This will break on windows! The whole point of using pathlib.Path objects is to avoid the problem of handling windows versus POSIX paths.  This is converting to a string type in POSIX format! Replace with `self.raw_path = self.raw_path.with_suffix(".mzml")` and then check where self.raw_path is used and where you might need to convert to a string.